torchbenchmark/util/backends/torchdynamo.py

```diff
@@ -0,0 +1,49 @@
+"""
+Support TorchDynamo(https://github.com/facebookresearch/torchdynamo) backends
+"""
+import argparse
+from typing import List
+
+class NullContext:
+    def __enter__(self):
+        pass
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        pass
+
+def help(fn):
+    return fn.__doc__
+
+def null_experiment(args, model_iter_fn, model, example_inputs):
+    """
+    A no-op experiment useful for making sure TorchBenchmark alone works properly.
+    """
+
+    return []
+
+def parse_torchdynamo_args(model: 'torchbenchmark.util.model.BenchmarkModel', dyamo_args: List[str]) -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
```

Comment:
Perhaps simplify this to just have one option:
`--backend="ofi"`
or 
`--torchdynamo="ofi"`

You can test with the "eager" backend.


Which would just call:
```
model.add_context(functools.partial(torchdynamo.optimize, args.backend))
```

Reply:
Thanks! I have addressed this comment.